blockbuster/src/com/codecool/emis/blockbuster/product/Product.java

```diff
@@ -0,0 +1,91 @@
+package com.codecool.emis.blockbuster.product;
+
+import java.time.LocalDate;
+
+public class Product {
+
+    private Platform platform;
+    private String title;
+    private LocalDate releaseDate;
+    private int dailyRentalCost;
+    private int originalPrice;
+    private ProductType productType;
+    private int howManyTimesRent = 0;
+    private boolean available = true;
+
```

Comment:
you could have another field here
private boolean blockbusterFavorite = false;
and that way you won't need two sets in the BlockBuster class